### ISSUE TITLE
Modifying `sys.path` leads to import errors

### ISSUE THREAD

**Issue by jonathanverner** [brython-dev/brython#679]
Adding a path to `sys.path` leads to not being able to import certain modules:

```python
>>> import sys
>>> sys.path.append('http://www.google.com')
>>> import os
Traceback (most recent call last):
AttributeError:object has no attribute info
>>> import os # The second and further imports lead to a different error

Exception:<TypeError: Cannot read property '$nat' of null>
>>> import turtle # some packages import just fine
>>>
```

Note that importing the module **before** modifying `sys.path` works just fine. Also note that if the added path points to a python module, this module imports just fine (though `os` and other system modules still don't import).  Testing on a local server, brython seems to request several files under the path (which are not there). E.g. if I append `http://localhost:7123/static/core/python/` to `sys.path`, the following requests are made

```
127.0.0.1 - - [2017-09-18 18:40:24] "GET /static/core/python/_frozen_importlib.js?v=1505752824903 HTTP/1.1" 404 1088 0.001479
127.0.0.1 - - [2017-09-18 18:40:24] "GET /static/core/python/_frozen_importlib.pyc.js?v=1505752824913 HTTP/1.1" 404 1092 0.001399
127.0.0.1 - - [2017-09-18 18:40:24] "GET /static/core/python/_frozen_importlib/__init__.pyc.js?v=1505752824963 HTTP/1.1" 404 1101 0.001492
127.0.0.1 - - [2017-09-18 18:40:25] "GET /static/core/python/_frozen_importlib.py?v=1505752825015 HTTP/1.1" 404 1088 0.001666
127.0.0.1 - - [2017-09-18 18:40:25] "GET /static/core/python/_frozen_importlib/__init__.py?v=1505752825067 HTTP/1.1" 404 1097 0.001740
```

**Comment by imnvr** [comment]
Paid a bounty for this here:
https://www.bountysource.com/issues/49462861-modifying-sys-path-leads-to-import-errors

**Comment by YujinYuz** [comment]
I tested this on the website just now at: http://brython.info/tests/console.html?lang=en

to try and replicate the issue.

But I didn't seem to get any errors.

```python
Brython 3.3.3 on Netscape 5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/60.0.3112.90 Safari/537.36 OPR/47.0.2631.80
>>> import sys
>>> sys.path.append('http://www.google.com')
>>> import os
>>> os.getcwd()
'http://brython.info/tests'
>>> 
```

May I know what did you do exactly?

**Comment by jonathanverner** [comment]
I suspect this was fixed by 668e1d0381... No idea why it didn't work yesterday, but I promise it didn't :-)
